standalone/test/integration_tests/api/integrationtest_scanner_api.cpp

```diff
@@ -152,7 +149,7 @@ TEST_F(ScannerAPITests, testStartFunctionality)
 
 TEST_F(ScannerAPITests, shouldReturnInvalidFutureWhenStartIsCalledSecondTime)
 {
-  setUpNiceScannerMock();
+  setUpScannerMock(false);
```

Comment:
Boolean arguments are realy confusing. The false can be interpreted as don't set up the scanner mock since the method name does not tell you what the argument is supposed to do.

Reply:
Ok. I will revert it.